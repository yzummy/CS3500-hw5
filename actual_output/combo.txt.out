Line 9: Arg 1 must be list
