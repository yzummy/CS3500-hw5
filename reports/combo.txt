Line 9: Arg 1 must be list				      |	 96.86
							      >	  2.17
							      >	(   3.14 4 "BoJack" TRUE )
							      >	FALSE
							      >	  1.18
