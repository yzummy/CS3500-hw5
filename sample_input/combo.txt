{
  A = list(3.14, -21, "BoJack", TRUE);
  x = 100 - A[[1]];
  print(x);
  w = 200 %% 2.71;
  print(w);
  A[[2]] = A[[2]] + 25;
  print(A);
  y = if ((x > w) | A[[4]])
      {
        cat(! w);
        z = (w ^ 2) / 4
      }
      else "Mr. Peanutbutter";
  print(y);
  quit()
}
