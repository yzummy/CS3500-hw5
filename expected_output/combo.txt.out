 96.86
  2.17
(   3.14 4 "BoJack" TRUE )
FALSE
  1.18
